rails-commenting.rb

```diff
@@ -7,24 +7,30 @@
 # FILE: app/controller/blog_posts_controller.rb
 
```

Comment:
The methods are explained very well in the section of the syllabus
https://github.com/learn-academy-2022-charlie/Syllabus/blob/main/rails/fullstack.md